packages/website/docs/examples/block-from-tiptap.md

```diff
@@ -0,0 +1,557 @@
+---
```

Comment:
This is the only demo I'm not really convinced of. It seems:

a) quite cumbersome atm to create a block from TipTap
b) I think the use-case doesn't make complete sense. My guess is that what most people would want to do, is use an existing TipTap node (reuse an existing extension). Here, we create a new Node entirely. I think for this use-case, the user should just create a BlockNote custom block directly. (i.e.: the proper use-case for this demo would involve re-using a tiptap node from an existing extension / package)